src/request.ts

```diff
@@ -452,19 +466,24 @@ class DatastoreRequest {
    *   const entities = data[0];
    * });
    */
-  get(keys, options?): Promise<EntityDataObj[]>;
-  get(keys, options?, callback?): void|Promise<EntityDataObj[]> {
-    if (is.fn(options)) {
-      callback = options;
-      options = {};
-    }
-
-    options = options || {};
+  get(keys: Entities,
+      options?: CreateReadStreamOptions): Promise<Entity|Stream>;
+  get(keys: Entities, callback: GetCallback): void;
+  get(keys: Entities, options: CreateReadStreamOptions,
+      callback: GetCallback): void;
+  get(keys: Entities, optionsOrCallback?: CreateReadStreamOptions|GetCallback,
+      cb?: GetCallback): void|Promise<Entity|Stream> {
+    const options =
```

Comment:
We aren't doing this null check in other places, so I don't think it's entirely needed.

Reply:
Yeah I agree, I'd prefer the consistency - the problem is that `typeof null` evalutes to `'object'` in JS (thank you C) and a test uses null to trigger options be set to an object literal:
```
it('should default options to an object', done => {
        request.get(keys, null, err => {  // 2nd arg 'null' is what's being used to trigger the options object correction logic
          assert.ifError(err);

          const spy = request.createReadStream.getCall(0);
          assert.deepStrictEqual(spy.args[1], {});
          done();
        });
      });
```
Was thinking we should actually add the null check in other places because this doesn't tend to be accounted for